tests/dateSelector.spec.ts

```diff
@@ -50,91 +48,69 @@ test('select the desired date in the calendar part II', async ({ page }) => {
-            case 'YYYY-MM-DD':
-                return `${year}-${month}-${day}`
-            case 'MM YYYY':
-                return `${month} ${year}`;
-            default:
-                throw new Error('The "expectedFormat" parameter is required. Please specify a valid date format.')
-        }
-    }
-
-    const newVisitPageDateFormat = formatDate(date, 'YYYY/MM/DD')
-    const ownerPageDateFormat = formatDate(date, 'YYYY-MM-DD')
-
-    console.log('Formatted Date on New Visit Page:', newVisitPageDateFormat);
-    console.log('Formatted Date on Owner Page:', ownerPageDateFormat);
+    const currentDateFormatted = `${year}-${month}-${day}`
+    await expect(page.locator('.mat-datepicker-input')).toHaveValue(`${year}/${month}/${day}`)
 
     //8. Type the description in the field, for example, "dermatologists visit" and click "Add Visit" button
-    await page.locator('#description').fill('veterinary internist visit')
+    await page.locator('#description').fill('dermatology visit')
     await page.getByRole('button', { name: 'Add Visit' }).click()
 
     //9. Add assertion that selected date of visit is displayed at the top of the list of visits 
     //for "Samantha" pet on the "Owner Information" page and is in the format "YYYY-MM-DD"
-    await expect(addVisitButtonOwnerPage.locator('app-visit-list tr td').first()).toHaveText(expectedDermatologyAppointmentDate)
+    await expect(petsAndVisitsRowOfSamatha.locator('app-visit-list tr td').first()).toHaveText(currentDateFormatted)
 
     //10. Add one more visit for "Samantha" pet by clicking "Add Visit" button
-    await addVisitButtonOwnerPage.getByText('Add Visit').click()
+    await petsAndVisitsRowOfSamatha.getByText('Add Visit').click()
     //11. Click on the calendar icon and select the date which is 45 days back from the current date
     await page.locator('.mdc-icon-button').click()
-
     date.setDate(date.getDate() - 45)
     // Get the year, month, and day for the massage appointment
-    const massageAppointmentYear = date.getFullYear();
-    const massageAppointmentMonth = date.toLocaleString('en-US', { month: '2-digit' })
-    const massageAppointmentDaySingleDigit = date.toLocaleString('en-US', { day: 'numeric' })
-    const massageAppointmentDayDoubleDigit = date.toLocaleString('en-US', { day: '2-digit' })
+    const previousAppointmentYear = date.getFullYear();
+    const previousAppointmentMonth = date.toLocaleString('en-US', { month: '2-digit' })
+    const previousAppointmentDayNumeric = date.toLocaleString('en-US', { day: 'numeric' })
+    const previousAppointmentDay2Digit = date.toLocaleString('en-US', { day: '2-digit' })
     // Format the expected appointment date as 'YYYY-MM-DD'
-    const expectedMassageAppointmentDate = `${massageAppointmentYear}-${massageAppointmentMonth}-${massageAppointmentDayDoubleDigit}`;
+    const expectedMassageAppointmentDate = `${previousAppointmentYear}-${previousAppointmentMonth}-${previousAppointmentDay2Digit}`;
```

Comment:
If you changed "expectedDermatologyAppointmentDate" to "currentDateFormatted", then for consistency "expectedMassageAppointmentDate" should also be changed.